client/cmd/bisonw-desktop/app_darwin.go

```diff
@@ -306,8 +306,6 @@ func mainCore() error {
 		}()
 	}
 
-	// Default to serving the web interface over TLS.
-	cfg.WebTLS = true
```

Comment:
I'm not sure why this setting would be forced. I do see there was [a tangentially related comment](https://github.com/decred/dcrdex/pull/2372#discussion_r1224785817) in the PR, and I'm wondering if it was misinterpreted. Hopefully @ukane-philemon can shed some light, but I don't see a reason to default to TLS on the desktop app, which will always be running localhost.